staff/acct/signat

```diff
@@ -15,60 +15,98 @@ from ocflib.ucb.groups import (
-    elif 'calnetUid' not in attrs:
-        print(red("CalNet UID not found."))
+        print(red("User not found."))
         return
 
-    show_groups_by_student_signat(int(attrs['calnetUid'][0]))
+    uid = attrs.get('calnetUid', [None])[0]
+    oid = attrs.get('callinkOid', [None])[0]
+
+    if uid and oid:
+        print(yellow("Both CalNet UID and CalLink OID are defined for " +
+            "this account."))
+        show_groups_by_student_signat(int(uid))
+        print()
+        show_signatories_for_group(int(oid))
+    elif uid:
+        show_groups_by_student_signat(int(uid))
+    elif oid:
+        show_signatories_for_group(int(oid))
+    else:
+        print(red("This account has no CalNet UID or CalLink OID."))
 
 def show_signatories_for_group(oid, signatories=None):
     """Show all signatories for a group."""
     if signatories is None:
         signatories = signatories_for_group(oid)
+    group = group_by_oid(oid)
 
-    if not signatories:
-        print(red("No signatories found."))
+    if not signatories and not group:
+        print(red("CalLink OID {} not found.".format(oid)))
         return
-
-    name = group_by_oid(oid)['name']
-    title = "{} ({})".format(name, oid)
-    columns = [(attrs['name'], uid) for (uid, attrs) in signatories.items()]
+    elif not signatories:
+        print(red("CalLink OID {} currently has no signatories.".format(oid)))
+        return
+    elif not group:
+        print(yellow(("CalLink OID {} has signatories but is currently not " +
+            "active on CalLink.").format(oid)))
+
+    if group:
+        if group['accounts']:
+            subtitle = "Group accounts: {}".format(', '.join(group['accounts']))
+        else:
+            subtitle = None
+        title = '{} ({})'.format(group['name'], oid)
+    else:
+        subtitle = None
+        title = "Unkown group ({})".format(oid)
+
+    columns = [(attrs['name'], uid) for uid, attrs in signatories.items()]
     headers = ('Signatory', 'UID')
 
-    print_table(title, columns, headers)
+    print_table(title, columns, headers, subtitle=subtitle)
 
 
 def show_groups_by_student_signat(uid, groups=None):
     """Show signatory status of a person on CalLink."""
+    name = directory.name_by_calnet_uid(uid)
+
+    if not name:
+        print(red("CalNet UID {} not found.").format(uid))
+        return
+
     if groups is None:
         groups = groups_by_student_signat(uid)
 
     if not groups:
-        print(red("No active student groups found."))
+        print(red("CalNet UID {} not a signatory of any active student " +
+            "groups.").format(uid))
         return
 
-    name = directory.name_by_calnet_uid(uid)
-    title = "{} ({})".format(name, uid)
-    columns = [(attrs['name'], uid) for (uid, attrs) in groups.items()]
-    headers = ('Group', 'OID')
+    title = '{} ({})'.format(name, uid)
+    columns = [(attrs['name'], ', '.join(attrs['accounts']) or 'n/a', oid) for
+            oid, attrs in groups.items()]
```

Comment:
Sorry, I just auto-linebreak after 79 characters. Feel free to apply coding standards.